TheWarWithin/RogueSubtlety.lua

```diff
@@ -283,15 +283,14 @@ spec:RegisterAuras( {
         duration = 3,
         max_stack = 1,
     },
-    perforated_veins_stack = {
+    perforated_veins = {
```

Comment:
Careful with this sort of change, as we can drift out of alignment with SimulationCraft.  It looks like this is okay, since `perforated_veins_stack` is not present in SimC any longer.

However, it looks like SimC has `perforated_veins` as well as `perforated_veins_counter` -- so we need to be sure those both are emulated in case both are referenced in the APL at some point.

Can you confirm in-game whether Perforated Veins has different spell IDs for the stacking counter vs. the fully charged aura that gets consumed?  I couldn't find a log using Perforated Veins quickly on WCL.

Reply:
Negative, it is a single buff that simply stacks to 4 then disappears on the 5th cast, starts over on the 6th. Similar to the serpentine rhythm thing from BM hunter a couple weeks ago. No name changes, no ID changes.

Maybe `copy = "perforated_veins_stack"` or something?

Reply:
If it disappears on the 5th cast, we need to be able to consume it (everywhere that is appropriate).

![image](https://github.com/user-attachments/assets/a81419b4-1bb8-42c6-be6c-f60653c78b8d)

If you Gloomblade 4x and use another CP generator (Fan?), does the buff go away on cast 5?

It's not incredibly significant, since it's not currently used for APL decision-making anyway (and doesn't change resources).